src/cli.rs

```diff
@@ -1148,9 +1148,13 @@ pub fn handle_cli_edge_app_command(command: &EdgeAppCommands) {
             generate_mock_data,
         } => {
             let secrets = if let Some(secret_pairs) = secrets {
-                secret_pairs.secrets.clone()
+                secret_pairs
```

Comment:
why does it need to be converted here? And I don't see any reason why secret pairs need to be of this type. Especially in exposed API.

Any specific reason?

Reply:
No specific reason actually. Was just quickly following the reference chain. Maybe overdone it :smile:

Reply:
I feel like it's a bit more natural for us not to expose those internal types they can be private to the module so that consumers don't have to care about it

Reply:
Actually, the exposed part is still the same. Are you sure we're exposing the new type? Maybe I'm overseeing something?

Reply:
EdgeAppServerValue::Str is exposed which is a private enum

Reply:
Done!